### PR TITLE
Create 'Create Todo' View

### PR DESCRIPTION
**Pull request by nickkhall**
**Components**:
  - Created `CreateTodoModal` component
  - Created `CreateTodoForm` component
  - Created `NoTodos` component

**Tests**:
  - Created test for `CreateTodoForm` component

----

*Current Screenshot of `NoTodos` component*:
screenshot from 2019-03-02 22-03-11

*Current screenshot of `CreateTodoModal` component*:
screenshot from 2019-03-02 22-03-19

### CODE REVIEW

File: client/actions/Modals/index.js [hunk 1]
```diff
@@ -10,24 +10,22 @@ import {
  * @param {string} modalName The name of the modal to set.
  * @param {!Object} modalProps The props to pass through the modal being set.
  */
-export const setModal = (modalName, modalProps) => dispatch => (
+export const setModal = (modalName, modalProps) => dispatch =>
```
Comment (nickhallph) [inline client/actions/Modals/index.js:13]: Move the returning `dispatch` to the next line

File: client/actions/Modals/index.js [hunk 2]
```diff
@@ -10,24 +10,22 @@ import {
...
-);
+  });
 
 /**
  * Closes a modal (by removing it from the stack).
  * If "force" is set to true, the modal stack will be cleared/emptied.
  * @param {boolean} force Whether or not to clear the stack completely.
  */
-export const closeModal = (force = false) => dispatch => (
+export const closeModal = (force = false) => dispatch =>
```
Comment (nickhallph) [inline client/actions/Modals/index.js:27]: Move the returning `dispatch` to the next line

**Review by nickhallph — CHANGES_REQUESTED** [review]
LGTM, requesting a few changes

File: client/actions/Modals/index.js (hunk 1 above)
Comment (nickkhall) [reply client/actions/Modals/index.js:13]: I did idiot

File: client/actions/Modals/index.js (hunk 2 above)
Comment (nickkhall) [reply client/actions/Modals/index.js:27]: You betcha!

**Review by nickhallph — APPROVED** [review]
agreed
LGTM

**Pull request merged**